python-package/lightgbm/dask.py

```diff
@@ -741,12 +1093,21 @@ def fit(
         y_shape="Dask Array, Dask DataFrame or Dask Series of shape = [n_samples]",
         sample_weight_shape="Dask Array, Dask DataFrame, Dask Series of shape = [n_samples] or None, optional (default=None)",
         init_score_shape="Dask Array, Dask DataFrame, Dask Series of shape = [n_samples] or None, optional (default=None)",
-        group_shape="Dask Array, Dask DataFrame, Dask Series of shape = [n_samples] or None, optional (default=None)"
+        group_shape="Dask Array, Dask DataFrame, Dask Series of shape = [n_samples] or None, optional (default=None)",
+        eval_sample_weight_shape='List of Dask Arrays, Dask DataFrames, Dask Series or None, optional (default=None)',
+        eval_init_score_shape='List of Dask Arrays, Dask DataFrames, Dask Series or None, optional (default=None)',
+        eval_group_shape='List of Dask Arrays, Dask DataFrames, Dask Series or None, optional (default=None)'
     )
 
-    # DaskLGBMClassifier does not support evaluation data, or early stopping
+    # DaskLGBMClassifier does not support group, eval_group, early_stopping_rounds.
     _base_doc = (_base_doc[:_base_doc.find('group :')]
-                 + _base_doc[_base_doc.find('verbose :'):])
+                 + _base_doc[_base_doc.find('group :'):])
+
+    _base_doc = (_base_doc[:_base_doc.find('eval_group :')]
+                 + _base_doc[_base_doc.find('eval_group :'):])
+
+    _base_doc = (_base_doc[:_base_doc.find('early_stopping_rounds :')]
+                 + _base_doc[_base_doc.find('early_stopping_rounds :'):])
```

Comment:
Something is wrong with these (and in all other places of this PR) string manipulations. They actually do nothing. Please take a look at the rendered docs for `DaskLGBMClassifier` for example:

![image](https://user-images.githubusercontent.com/25141164/119044202-92a4cd00-b9c2-11eb-8ec0-336e1ca64290.png)

Reply:
Sure yeah I'll figure it out, thanks for pointing this out

Reply:
Addressed in ce9de34